### ISSUE TITLE
Kitematic fails to download new container

### ISSUE THREAD

**Issue by worxco** [docker/kitematic#1759]
### Expected behavior
When I select a 'recommended' container to install, it should install.

### Actual behavior
When I select 'jenkins' or 'mincraft' containers to install, I get a network error as follows:

Network timed out while trying to connect to https://index.docker.io/v1/repositories/kitematic/minecraft/images. You may want to check your internet connection or if you are behind a proxy.

### Information about the Issue
OS X 10.11.5 / 8G memory / Macbook Air / 61% of disk used (512G)
I restarted Kitematic and still get the same results.
I am not behind a proxy.
My internet connection is fine.
I CAN get to https://index.docker.io/v1/repositories/kitematic/minecraft/images in my browser.

The result of the browser (Chrome) query is:
[{"checksum": "", "id": "74315596d98926af841aa3f62d122bbbb069d561b2ac43393873928e7a58b2fc"}, {"checksum": "", "id": "081520d74b6d468989ea1f17663db4ef5fbd9b06be1601a9c2ac8e045b90841d"}, {"checksum": "", "id": "17651f8c81b433e1309545f5286fc8cfdc618ea1bc2036c5199dac7bc6ad3fc2"}, {"checksum": "", "id": "67270782efcf1e30769009591dff4c605772be1224ba492833aac431df4128db"}, {"checksum": "", "id": "c4af60422c54a5737b652a8c66fca088a14d00875b0e6d2169dc90e23c214267"}, {"checksum": "", "id": "b5934c2c168b9eb244ca25eaf60c50b1fa9c591f19ffba67faf9c227cb946199"}, {"checksum": "", "id": "838266c57b6ec9c34a311ab6bc1d99889bfdd30b6c984d9b74a04b299535dd88"}, {"checksum": "", "id": "41c51fdfbcbda3dd4fbd19c32ab0084f5f9c3adc3b600c76f7ede608b9a6005b"}, {"checksum": "", "id": "114daceaa5bd610ce7dbce042f358a4c3592e0babf8642caf121f968550aa703"}, {"checksum": "", "id": "32f9078f3f3ffabcc524aa6c6a97695d6165404ab7f0381aab86336342e37b54"}, {"checksum": "", "id": "c6b75a24bf01c4e2e72373767b2f4b673a0928eef72ca4552f958c4e9785f7e1"}, {"checksum": "", "id": "3d6fe2e33ca75e979fa2dbcaa2aed12512b1e806b6268d43e439090d9edf0be7"}, {"checksum": "", "id": "ade1f53e6018de3671bb6170f3de677064c043feecf7370cac57ee4a001e28b8"}, {"checksum": "", "id": "ca686696237b92d41ea75152ec4f2e3ab7ab7fd79d9aa51f4f9ac1098c6bd7a9"}, {"checksum": "", "id": "a8575f10029ecfcca24698be50c8b94d4ee229c0c59e5f65c9da53e08059e602"}, {"checksum": "", "id": "bfda257a9290a54e70ebee5f01aa1955109e6d2b0a539de9097fd9634162b015"}, {"checksum": "", "id": "c90d655b99b2ec5b7e94d38c87f92dce015c17a313caeaae0e980d9b9bed8444"}, {"checksum": "", "id": "30d39e59ffe287f29a41a3f8bd70734afc8728329e3289945cbdc5bbf07cd980"}, {"checksum": "", "id": "511136ea3c5a64f264b78b5433614aec563103b4d4702f3ba7d4d2698e22c158"}]

### Steps to reproduce the behavior

  1. Start Kitematic
  2. Select Jenkins
  3. Wait....
  4. See this error:
Network timed out while trying to connect to https://index.docker.io/v1/repositories/library/jenkins/images. You may want to check your internet connection or if you are behind a proxy.
  5. Use the previous URL in your browser and watch the json appear - go figure....

**Comment by FrenchBen** [comment]
Are you behind a proxy? Can you share details about your setup?

I've created 5 jenkins containers on 3 different machines

**Comment by worxco** [comment]
I am not behind a proxy - but I am confused.
I got to the house and everything works fine there...
So, it sounds like a proxy, but it is just NAT and Firewall....

I'll check and see if there is a PORT that I need to open or something like that...
Obviously, it is NOT Kitematic - more on the order of loose screw between seat and keyboard.

**Comment by FrenchBen** [comment]
:) thank you for reporting that Kitematic isn't the issue. 
I would look into the setup of NAT + Firewall. 
We have some info on 'proxy' setup here:
https://github.com/docker/kitematic/wiki/Common-Proxy-Issues-&-Fixes

Essentially you can ssh into the VM and try to ping the outside world from within. I'm guessing it can't for some reason:
`docker-machine ssh default` will get you in the VM

**Comment by worxco** [comment]
While I didn't "solve" the problem as in discovering the problem in detail and correct, I did figure out that it has something to do with my wireless router. And ONLY the wireless portion.
I have both wireless and wired connection to the router. I disconnected the wireless connection and it works just fine.

Wireless from ANY box in the office is failing.
I just had one box that had 2 connections and the wired side works.

I will dig more and report back in case the search engines offer this up to some poor soul with a similar issue.
